app/jobs/open_prison_transfer_job.rb

```diff
@@ -18,7 +18,21 @@ def perform(movement_json)
     return unless PrisonService.open_prison?(offender.prison_id)
 
     Rails.logger.info("[MOVEMENT] Processing move to open prison for #{offender.offender_no}")
-    send_email(offender, movement) if offender.ldu_email_address.present?
+
+    if offender.prison_id == PrisonService::PRESCOED_CODE && offender.welsh_offender
+      return unless offender.indeterminate_sentence?
+
+      if offender.ldu_email_address.present?
+        prescoed_send_email(offender, movement)
+
+        EmailHistory.create! nomis_offender_id: offender.offender_no, name: offender.ldu_name,
+                             email: offender.ldu_email_address,
+                             event: EmailHistory::WELSH_TRANSFER_TO_PRESCOED,
+                             prison: offender.prison_id
```

Comment:
I think the `EmailHistory` event name would ideally be non-Prescoed specific.

The email itself is really about the LDU needing to allocate a COM to an offender who's moved into an open prison where OMIC rules apply. The email template itself makes no reference to Prescoed.

The same goes for the method name `prescoed_send_email` – I think it could be more generic, since it actually sends the email called `omic_open_prison_community_allocation`.

Finally, I think it might be nice to keep the `EmailHistory.create!` in the same method as the call to `CommunityMailer.omic_open_prison_community_allocation` – so it all happens in the same place, and it's really clear to see that the EmailHistory record directly relates to the email being sent.